src/main/java/net/dv8tion/jda/api/components/selections/StringSelectMenu.java

```diff
@@ -143,6 +152,7 @@ static Builder create(@Nonnull String customId)
     class Builder extends SelectMenu.Builder<StringSelectMenu, StringSelectMenu.Builder>
     {
         private final List<SelectOption> options = new ArrayList<>();
+        private boolean required = true;
```

Comment:
Let's just not define it by default, since discord picks the proper default in the right context.